src/Files.App/Services/Settings/GeneralSettingsService.cs

```diff
@@ -263,6 +263,12 @@ public FileNameConflictResolveOptionType ConflictsResolveOption
 			set => Set(value);
 		}
 
+		public bool ShowSettingsButtonOnAddressToolbar
```

Comment:
I had an idea to invert the setting to "Display settings button on the sidebar" and to set the value to true.

Reply:
Sure